src/dyad/stream/dyad_stream_core.cpp

```diff
@@ -122,6 +122,7 @@ void dyad_stream_core::init (const dyad_params &p)
                               p.m_cons_managed_path.c_str (),
                               static_cast<dyad_dtl_mode_t> (p.m_dtl_mode),
                               &m_ctx);
+    m_ctx->use_fs_locks = false; // this is to disable checking for fs lock based logic.
```

Comment:
Add a TODO note saying that we will implement a lock file based synchronization scheme.

Reply:
FIXED